mangum/adapter.py

```diff
@@ -5,6 +5,7 @@
 
 from mangum.protocols import HTTPCycle, LifespanCycle
 from mangum.handlers import ALB, HTTPGateway, APIGateway, LambdaAtEdge
+from mangum.handlers.utils import DEFAULT_TEXT_MIME_TYPES
```

Comment:
Please move the `DEFAULT_TEXT_MIME_TYPES` to this file now that it is configurable - shouldn't need to reference it in the handlers or related utility functions.

Reply:
Done. See my comment re:backwards compatibility.